src/daily-dropoffs-report.tsx

```diff
@@ -0,0 +1,376 @@
+  const innerHeight = height - margin.top - margin.bottom;
+  const ref = useRef(null as SVGSVGElement | null);
+
+  const x = scalePoint()
+    .domain(STEPS.map(({ key }) => key))
+    .range([0, innerWidth]);
+
+  const y = scaleLinear()
+    .domain([0, max(data || [], (d) => d.welcome) as number])
+    .range([innerHeight, 0]);
+
+  const line = d3Line()
+    .x((d) => x((d as unknown as StepCount).step) as number)
+    .y((d) => y((d as unknown as StepCount).count) as number) as (s: StepCount[]) => string;
+
+  return (
+    <svg ref={ref} height={height} width={width}>
+      <Axis axis={axisLeft(y)} transform={`translate(${margin.left}, ${margin.top})`} />
+      <Axis
+        axis={axisBottom(x).tickFormat(stepToTitle as (s: string) => string)}
+        transform={`translate(${margin.left}, ${margin.top + innerHeight})`}
+        className="x-axis"
+        rotateLabels={width < 700}
+      />
+      <g transform={`translate(${margin.left}, ${margin.top})`}>
+        {(data || []).map((row) => (
+          <path
+            d={line(toStepCounts(row))}
+            fill="none"
+            stroke={color(row.issuer)}
+            stroke-width="1"
+            onMouseEnter={() =>
+              ref.current &&
+              select(ref.current)
+                .selectAll(".dots")
+                .attr("hidden", function () {
+                  const elem = this as SVGGElement;
+                  return elem.dataset.issuer === row.issuer ? null : false;
+                })
+            }
+          />
+        ))}
+      </g>
+      <g transform={`translate(${margin.left}, ${margin.top})`}>
+        {(data || []).map((row) => (
+          <g className="dots" hidden data-issuer={row.issuer}>
+            {toStepCounts(row).map(({ step, count }, idx, stepCounts) => {
+              let comparedToFirst = 0;
+              if (idx > 0) {
+                comparedToFirst = count / stepCounts[0].count;
+              }
+
+              return (
+                <>
+                  <circle cx={x(step)} cy={y(count)} r="3" fill={color(row.issuer)} />
+                  <text x={x(step)} y={y(count)} font-size="12" dx="3" dy="-3">
+                    <tspan x={x(step)}>{formatWithCommas(count)}</tspan>
+                    {idx > 0 && (
+                      <tspan x={x(step)} dy="1.2em">
+                        (${formatAsPercent(comparedToFirst)})
+                      </tspan>
+                    )}
+                  </text>
+                </>
+              );
+            })}
+          </g>
+        ))}
+      </g>
+    </svg>
+  );
+}
+
+function DailyDropffsReport(): VNode {
+  const ref = useRef(null as HTMLDivElement | null);
+  const [width, setWidth] = useState(undefined as number | undefined);
+  const { setAgencies } = useContext(AgenciesContext);
+  const { start, finish, agency, env } = useContext(ReportFilterContext);
+
+  const { data } = useQuery(`${start.valueOf()}-${finish.valueOf()}`, () =>
```

Comment:
These cache keys are global, so there's a potential for cache reuse between this report and daily auths data fetching. Could add a simple prefix to create a namespace, e.g.

```suggestion
  const { data } = useQuery(`dropoffs/${start.valueOf()}-${finish.valueOf()}`, () =>
```